app/controllers/prior_authority/steps/authority_value_controller.rb

```diff
@@ -1,16 +1,16 @@
 module PriorAuthority
   module Steps
     class AuthorityValueController < BaseController
-      def edit
-        @authority_threshold = current_application.prison_law? ? 500 : 100
+      before_action :set_authority_threshold
 
+      def edit
         @form_object = AuthorityValueForm.build(
           current_application
         )
       end
 
       def update
-        @form_object = current_application.becomes(AuthorityValueForm)
+        @form_object = AuthorityValueForm.new(application: current_application, record: current_application)
```

Comment:
I think this could be marginally more compact/standard by using `AuthorityValueForm.build(current_application)`